iset.mm

```diff
@@ -66451,6 +66628,39 @@ the first case of his notation (simple exponentiation) and subscript it
       UEUFUGUHUIUJUK $.
   $}
 
+  ${
+    $d A f $.  $d B f $.  $d C f $.  $d D f $.
+    en2prd.1 $e |- ( ph -> A e. V ) $.
+    en2prd.2 $e |- ( ph -> B e. W ) $.
+    en2prd.3 $e |- ( ph -> C e. X ) $.
+    en2prd.4 $e |- ( ph -> D e. Y ) $.
+    en2prd.5 $e |- ( ph -> A =/= B ) $.
+    en2prd.6 $e |- ( ph -> C =/= D ) $.
+    $( Two unordered pairs are equinumerous.  (Contributed by BTernaryTau,
```

Comment:
"Two proper unordered pairs ..."

Reply:
Not sure I have an opinion about this one, but OK.